src/geshi/lisp.php

```diff
@@ -52,30 +52,301 @@
     'ESCAPE_CHAR' => '\\',
     'KEYWORDS' => array(
         1 => array(
-            'not','defun','princ','when',
-            'eval','apply','funcall','quote','identity','function',
-            'complement','backquote','lambda','set','setq','setf',
-            'defmacro','gensym','make','symbol','intern',
-            'name','value','plist','get',
-            'getf','putprop','remprop','hash','array','aref',
-            'car','cdr','caar','cadr','cdar','cddr','caaar','caadr','cadar',
-            'caddr','cdaar','cdadr','cddar','cdddr','caaaar','caaadr',
-            'caadar','caaddr','cadaar','cadadr','caddar','cadddr',
-            'cdaaar','cdaadr','cdadar','cdaddr','cddaar','cddadr',
-            'cdddar','cddddr','cons','list','append','reverse','last','nth',
-            'nthcdr','member','assoc','subst','sublis','nsubst',
-            'nsublis','remove','length',
-            'mapc','mapcar','mapl','maplist','mapcan','mapcon','rplaca',
-            'rplacd','nconc','delete','atom','symbolp','numberp',
-            'boundp','null','listp','consp','minusp','zerop','plusp',
-            'evenp','oddp','eq','eql','equal','cond','case','and','or',
-            'let','l','if','prog','prog1','prog2','progn','go','return',
-            'do','dolist','dotimes','catch','throw','error','cerror','break',
-            'continue','errset','baktrace','evalhook','truncate','float',
-            'rem','min','max','abs','sin','cos','tan','expt','exp','sqrt',
-            'random','logand','logior','logxor','lognot','bignums','logeqv',
-            'lognand','lognor','logorc2','logtest','logbitp','logcount',
-            'integer','nil','parse-integer','make-list','print','write'
+            '*', '**', '***', '+', '++', '+++', '-', '/', '//', '///',
+            '/=', '1+', '1-','<', '<=', '=', '>', '>=', 'abort', 'abs',
```

Comment:
Operators are no keywords.

Please check also for other langcheck issues and repair them. TIA.